docs/configuration.md

```diff
@@ -16,19 +16,19 @@ For configuration, opsdroid uses a single YAML file named `configuration.yaml`.
                 `C:\Documents and Settings\<User>\Application Data\opsdroid\opsdroid`
 - System `/etc/opsdroid/configuration.yaml` (*nix only)
 
-_Note: If no file named `configuration.yaml` can be found on one of these folders one will be created for you taken from the [example configuration file](https://github.com/opsdroid/opsdroid/blob/master/opsdroid/configuration/example_configuration.yaml)_
+_Note: If no file named `configuration.yaml` can be found on one of these folders, one will be created for you taken from the [example configuration file](https://github.com/opsdroid/opsdroid/blob/master/opsdroid/configuration/example_configuration.yaml)_
 
-If you are using one of the default locations you can run the command `opsdroid config edit` to open the configuration with your favorite editor (taken from the environment variable `EDITOR`) or the default editor vim.
+Suppose you are using one of the default locations. In that case, you can run the command `opsdroid config edit` to open the configuration with your favourite editor (taken from the environment variable `EDITOR`) or the default editor vim.
```

Comment:
LOL. British vs US English, "favourite" vs "favorite" - another battle in the spelling wars.